packages/transformers/svg/src/SVGTransformer.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+import nullthrows from 'nullthrows';
+import semver from 'semver';
+import collectDependencies from './dependencies';
+import parse from 'posthtml-parser';
+import render from 'posthtml-render';
```

Comment:
We should probably use an actual XML parser for SVG, not an HTML parser. I think there are some subtle differences.

Reply:
Despite the name it is a HTML _and_ XML tool (as is htmlparser2).

Hadn't thought of using it initially (hence JSDOM), but it does the job here and has the benefit of already being used.